pages/roadmap-test/[...slug].tsx

```diff
@@ -1,49 +0,0 @@
-import type { InferGetServerSidePropsType } from 'next';
```

Comment:
we should leave these. I was working on input to allow devs to encode json necessary to make debugging and development faster.

Reply:
I didn't get that. Were you planning on using the `pages/roadmap-test`? That's an endpoint I created to do some testing but wasn't using for anything else. Note that this is not the standard `pages/roadmap`.

Reply:
@SgtPooki I didn't get that. Were you planning on using the pages/roadmap-test? That's an endpoint I created to do some testing but wasn't using for anything else. Note that this is not the standard pages/roadmap.

Reply:
I also created those pages, maybe I didn't push my changes up? if those are yours, I'm fine with removing them